frontend/views/containers/sidebar/GroupsList.vue

```diff
@@ -1,70 +1,50 @@
 <template lang='pug'>
-div(v-if='groupsByName.length')
-  menu-parent(@select='handleMenuSelect')
-    menu-trigger.c-trigger(:class="`has-text-${isDarkTheme ? 'white' : 'dark'}`")
-      avatar.c-avatar(
-        src='/assets/images/default-avatar.png'
-        :alt='groupSettings.groupName'
-        :blobURL='groupSettings.groupPicture'
-      )
-      | {{ groupSettings.groupName }}
-      i.icon-angle-down
-
-    menu-content
-      menu-header
-        i18n Your Groups
-      ul
-        menu-item(
-          v-for='(group, index) in groupsByName'
-          :key='`group-${index}`'
-          tag='button'
-          :item-id='group.contractID'
-          :is-active='currentGroupId === group.contractID'
-          :has-divider='index === groupsByName.length - 1'
+ul.c-group-list(v-if='groupsByName.length')
+  li.c-group-list-item.group-badge(
+    v-for='(group, index) in groupsByName'
+    :key='`group-${index}`'
+    tag='button'
+    :class="{ 'is-active': currentGroupId === group.contractID}"
+  )
+    tooltip(
+      direction="right"
+      :text='group.groupName'
+    )
+      button.is-unstyled(@click="handleMenuSelect(group.contractID)")
+        avatar.c-avatar(
+          src='/assets/images/default-avatar.png'
+          :blobURL='group.groupPicture'
```

Comment:
@pieer the cause of the missing group picture is caused by how "groupsByName" is constructed.

Before we accessed `groupSettings` that brings all the settings of a group. Now we just consume `groupsByName` that doesn't bring the `groupPicture`. We need to change `state.js`  so it also returns the group settings so we can access it as `group.settings.groupPicture`.

Reply:
I see! ok, it's fix now